src/get-env-variables.js

```diff
@@ -0,0 +1,60 @@
+const exec = require('child_process').exec
+// import spawn from 'spawn-command-with-kill'
+import {assign, clone} from 'lodash'
+import managePath from 'manage-path'
+import {sync as findUpSync} from 'find-up'
+
+const NON_ERROR = 0
+
+export default getEnv
+
+function getEnv() {
+  return new Promise((resolve, reject) => {
+    getNpmEnvVariables()
+      .then(npmVars => {
+        const env = assign(clone(process.env), npmVars)
+        const alterPath = managePath(env)
+        const npmBin = findUpSync('node_modules/.bin')
+        if (npmBin) {
+          alterPath.unshift(npmBin)
+        }
+        resolve(env)
+      })
+      .catch(reject)
+  })
+}
+
+function getNpmEnvVariables() {
+  let child
+  return new Promise((resolve, reject) => {
+    let output = ''
+    child = exec('npm run env')
+
+    child.stdout.on('data', chunk => output += chunk)
+
+    // NOTE: not sure what needs to happen with error logging
+    child.stderr.on('data', data => console.log(`stderr: ${data}`))
+
+    child.on('close', code => {
+      if (code === NON_ERROR) {
+        let npmVariables = parseNpmRunEnvOutput(output)
+        resolve(npmVariables)
+      } else {
+        // NOTE: again, not sure how we want to handle errors being logged
+        reject('npm run env exited with status code:', code)
+      }
+    })
+  })
+}
+
+function parseNpmRunEnvOutput (output) {
```

Comment:
Do you have a boilerplate for the basic OSS projects? Common things like build scripts + testing (I assume Jest for both node and react) + lint and prettier on precommit, CI config... Possibly documentation/guides for bots like LGTM or automatic issue expiration. Maybe some other cool things I don't know about that are common for OSS :)
I imagine it would be easier to base all of the new OSS and then update them later on via an upstream. Or you do the projects usually differ too much to abstract all of the common things.
Would be cool to have some kind of 'create-oss-app' I think. Or maybe there are some that I don't know about?